mne/io/edf/edf.py

```diff
@@ -1354,6 +1355,9 @@ def _read_annotations_edf(annotations):
             triggers = re.findall(pat, annot_file.read())
     else:
         tals = bytearray()
+        annotations = np.array(annotations) # To make sure that annotations is a ndarray
```

Comment:
I would just do `annotations = np.atleast_2d(annotations)` (or below `if annotations.ndim == 1: annotations = annotations[np.newaxis]` rather than going through `expand_dims`)

Reply:
Great remark, glad I learned a new numpy method today :+1: ! ! I will insert this change.